src/libraries/Common/src/Interop/Unix/System.Net.Security.Native/Interop.Initialization.cs

```diff
@@ -24,11 +24,6 @@ static SslInitializer()
 
             //Call ssl specific initializer
             Ssl.EnsureLibSslInitialized();
```

Comment:
Is is redundant? After line23, this has already ran: https://github.com/dotnet/runtime/blob/3806e15595632e4bc8e4c4fb4cde12163006eca9/src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Initialization.cs#L34